ingest/src/main/java/com/connexta/ingest/exceptions/StorageException.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.ingest.exceptions;
+
+public class StorageException extends RuntimeException {
```

Comment:
✏️Add the throws to the StorageAdaptor interface and add javadoc to that method.